src/views/viewCommands.ts

```diff
@@ -481,6 +485,16 @@ export class ViewCommands implements Disposable {
 		return RemoteActions.add(getNodeRepoPath(node));
 	}
 
+	@log()
+	private async addPullRequestRemote(node: ViewNode, pr: PullRequest, repo: Repository) {
+		const identity = getRepositoryIdentityForPullRequest(pr);
+		if (identity.remote?.url == null) return;
+		await repo.git
+			.remotes()
+			.addRemoteWithResult?.(identity.provider.repoDomain, identity.remote.url, { fetch: true });
```

Comment:
```suggestion
			.addRemote?.(identity.provider.repoDomain, identity.remote.url, { fetch: true });
```

Since we aren't returning the remote we don't need to wait for it